package/MDAnalysis/coordinates/DLPoly.py

```diff
@@ -125,17 +110,17 @@ def _read_first_frame(self):
             if has_forces:
                 forces = forces[order]
 
-        ts = self.ts = self._Timestep(self.n_atoms,
-                                      velocities=has_vels,
-                                      forces=has_forces,
-                                      **self._ts_kwargs)
+        ts = self.ts = base.Timestep(self.n_atoms,
+                                     velocities=has_vels,
+                                     forces=has_forces,
+                                     **self._ts_kwargs)
         ts._pos = coords
         if has_vels:
             ts._velocities = velocities
         if has_forces:
             ts._forces = forces
         if not imcon == 0:
-            ts._unitcell = unitcell
+            ts.triclinic_dimensions = unitcell
```

Comment:
Why add `Timestep.triclinic_dimensions` ? Is this the "original" unitcell – and if so, why not keep it as a private `_unitcell`. The user should not need to worry about box representations. We simply provide `ts.dimensions` and that's it.